src/OSPSuite.Core/Chart/PredictedVsObservedChart.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Domain.UnitSystem;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Core.Chart
+{
+   public abstract class PredictedVsObservedChart : AnalysisChart
+   {
+      private readonly Cache<float, IEnumerable<DataRepository>> _deviationRepositoryCache = new Cache<float, IEnumerable<DataRepository>>(onMissingKey: x => null);
```

Comment:
is there a reason why you are not storing an IReadOnlyList here?

Reply:
I need to know for each foldValue, if there was a data repository created. That's how I can tell the difference between what the chart is supposed to have, vs what it already has.

Reply:
oh, I see what you mean. forget my comment. above